src/pages/products/Products.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { EditableField } from '../../components/editableField';
+import { server } from '../../services';
+import './products.scss';
+
+const Product = (title, id) => (
+  <li
+    key={id}
+    className="product"
+  >
+    <div className="description">
+      <div className="setting">
+        <span>edit</span>
+        <span>X</span>
+      </div>
+      <Link to={`/products/:${id}`} className="img">To Product</Link>
+    </div>
+    <EditableField type="textarea" val={title} />
+  </li>
+);
```

Comment:
http://i.imgur.com/lxv9poT.png. There should be an icon for empty products and image for products with `image` field